app/utils/toke.py

```diff
@@ -10,7 +10,7 @@
 # from piazza_api import Piazza
 
 # EMPLOY CONSTANTS USED THROUGHOUT THE FILE AND CAN BE IMPORTED
-sp = spacy.load('en_core_web_lg') #TODO This is for speed
+sp = spacy.load('en_core_web_sm') #TODO This is for speed
```

Comment:
Could this be changed back? Does not impact functionality too much but will slightly change search results on specific search items.

Reply:
Youcan use sm for testing in your local place, but please ensure that the large package is used in deployment. As magd says, change it back and push it. :)

Reply:
@oscarso2000 We should eventually evaluate the impact this has on the performance as the smaller would be better if it could be used.

Reply:
okay sure got it!